libraries/AC_Avoidance/AP_OADatabase.cpp

```diff
@@ -141,23 +156,50 @@ void AP_OADatabase::update()
     database_items_remove_all_expired();
 }
 
+// get number of items in the database
+uint16_t AP_OADatabase::database_count() const
+{
+    if (_tracked_obs_enable) {
+        return _database.count + _database_tracked.count;
+    }
+
+    return _database.count;
+}
+
+// fetch an item in database. Undefined result when i >= _database.count + _database_tracked.count.
+float AP_OADatabase::get_item_pos_radius(uint32_t i, Vector3f& pos) const
```

Comment:
I'd prefer if this method returned both position and radius via the arguments instead of returning the radius and passing the position via the arguments.

Reply:
maybe call the method "get_item_pos_and_radius" or "get_item_position_and_radius"